doc/ref_model/chapters/glossary.md

```diff
@@ -26,8 +27,8 @@ To help guide the reader, this glossary provides an introduction to the terminol
 - **Network Service (NS)**: composition of **Network Function**(s) and/or **Network Service**(s), defined by its functional and behavioural specification, including the service lifecycle.
 - **Virtual Network Function (VNF)**: a software implementation of a **Network Function**, capable of running on the **NFVi**.
   - **VNF**s are built from one or more VNF Components (**VNFC**) and, in most cases,  the VNFC is hosted on a single VM or Container.
-- **Cloud-native (containerised) Network Function (CNF)**: **VNF** with a full adherence to cloud native principles, or a **VNF** that is transitioning to cloud native.
-  >_*Note:*_ A containerised **VNF** is microservices-oriented, to increase agility and maintainability, and can be dynamically orchestrated and managed to optimize resource utilization. The containers run on an application orchestration layer, not directly on the underlaying infrastructure itself. the containers themselves may be Linux, Docker or other similar container technology.
+- **Cloud-native (containerised) Network Function (CNF)**: An implementation of a Virtual Network Function (**VNF**, as defined by ETSI GS NFV 003) that adheres to the CNCF Cloud Native Definition or a **VNF** that is transitioning to cloud native.
```

Comment:
Suggest removal of `(containerised)` as there can/will be VM-based CNFs and also serverless to consider.  Also, once the TUG has agreed on the definition (currently focussed on ALT4.5/4.7 here: https://docs.google.com/document/d/1LO6G4sWHJKKW77z7gj2Aqt3IF3dRJL5q6Q1wXI4Kj8g/edit?pli=1#heading=h.tpyt1n9bctae

Reply:
OK. Let's remove "containerised" here. 
Since the TUG has not yet agreed on a new definition, I suggest to wait before trying to synchronize further.
Also I think we should consider carefully about the part of the definition saying that a CNF is an implementation of a VNF, which I think is very important in our context, because otherwise a NFVO and VNFM will not be able to manage it.

Reply:
done